docs/QUICK_START.md

```diff
@@ -0,0 +1,44 @@
+# Quick Start Guide
+
+Thanks for supporting AmpliPi, we hope you love your new unit!
+
+## Installation
+
+We suggest making the speaker connections before connecting AmpliPi to power.
+
+### Back Panel
+
+![Backpanel](imgs/backpanel.jpg)
+
+### Speakers
+
+Each of AmpliPi's 6 zones can drive a 4-8 Ohm speaker pair, using the provided Pheonix connectors. Here is what the basic wiring diagram for a zone looks like:
+
+![Speaker-Zone Diagram](imgs/zone_speaker_connection.png)
+
+AmpliPi's amplifiers are capable of driving 4-8 Ohm speaker loads, in a stereo configuration, Here is what a typical speaker connection, using CL2 rated 14-AWG speaker wire and the Pheonix connectors, looks like:
```

Comment:
```suggestion
AmpliPi's amplifiers are capable of driving 4-8 Ohm speaker loads, in a stereo configuration. Here is what a typical speaker connection, using CL2 rated 14-AWG speaker wire and the Pheonix connectors, looks like:
```
Is the first sentence even required though, since it was just mentioned in the previous sentence?